### ISSUE TITLE
One or more errors occurred. (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width)

### ISSUE THREAD

**Issue by GooseWithAKnife** [RatScanner/RatScanner#591]
**Error**
One or more errors occurred. (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width)
```
System.AggregateException: One or more errors occurred. (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width)
 ---> OpenCvSharp.OpenCVException: _img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width
   at OpenCvSharp.Internal.NativeMethods.<>c.<.cctor>b__1682_0(ErrorCode status, String funcName, String errMsg, String fileName, Int32 line, IntPtr userData)
   at OpenCvSharp.Internal.NativeMethods.imgproc_matchTemplate(IntPtr image, IntPtr templ, IntPtr result, Int32 method, IntPtr mask)
   at OpenCvSharp.Cv2.MatchTemplate(InputArray image, InputArray templ, OutputArray result, TemplateMatchModes method, InputArray mask)
   at RatEye.Processing.Icon.<>c__DisplayClass42_0.<TemplateMatchSub>b__0(KeyValuePair`2 icon)
   at System.Threading.Tas
```
<details>
<summary>Log</summary>

```
[20:46:56.1222215] > [Info]  ----- RatScanner 3.6.4 -----
[20:46:56.1225383] > [Info]  Screen Info: 1920x1080 at 150%
[20:46:56.1225409] > [Info]  Setting temporary default item...
[20:46:56.1234329] > [Info]  Initializing tarkov tracker database
[20:46:56.1235291] > [Info]  Initializing hotkey manager...
[20:46:56.1271608] > [Info]  UI Ready!
[20:46:57.1392774] > [Info]  Checking for updates...
[20:46:57.1407758] > [Info]  Loading resource "RSClientVersion"...
[20:46:57.9505577] > [Info]  Loading item data...
[20:46:59.6601468] > [Info]  Loading price data...
[20:47:00.6808661] > [Info]  Loading TarkovTracker data...
[20:47:00.6808739] > [Info]  Initializing RatEye...
[20:47:01.0336139] > [Info]  Setting up timer routines...
[20:47:01.0336322] > [Info]  Setting default item...
[20:47:01.0351993] > [Info]  Enabling hotkeys...
[20:47:01.0352369] > [Info]  Ready!
[20:48:11.1661331] > [Info]  Saving config...
[20:48:11.1969190] > [Info]  Config saved!
[20:48:18.2854658] > [Info]  Saving config...
[20:48:18.3183475] > [Info]  Config saved!
[20:48:24.0858563] > [Error] One or more errors occurred. (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width)
 -------------------- 
 System.AggregateException: One or more errors occurred. (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width) (_img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width)
 ---> OpenCvSharp.OpenCVException: _img.size().height <= _templ.size().height && _img.size().width <= _templ.size().width
   at OpenCvSharp.Internal.NativeMethods.<>c.<.cctor>b__1682_0(ErrorCode status, String funcName, String errMsg, String fileName, Int32 line, IntPtr userData)
   at OpenCvSharp.Internal.NativeMethods.imgproc_matchTemplate(IntPtr image, IntPtr templ, IntPtr result, Int32 method, IntPtr mask)
   at OpenCvSharp.Cv2.MatchTemplate(InputArray image, InputArray templ, OutputArray result, TemplateMatchModes method, InputArray mask)
   at RatEye.Processing.Icon.<>c__DisplayClass42_0.<TemplateMatchSub>b__0(KeyValuePair`2 icon)
   at System.Threading.Tasks.Parallel.<>c__DisplayClass44_0`2.<PartitionerForEachWorker>b__1(IEnumerator& partitionState, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.<>c__DisplayClass44_0`2.<PartitionerForEachWorker>b__1(IEnumerator& partitionState, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
   at System.Threading.Tasks.TaskReplicator.Replica.Execute()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.TaskReplicator.Run[TState](ReplicatableUserAction`1 a
```
</details>

**Comment by MilanDierick** [comment]
+1

**Comment by Blightbuster** [comment]
Fixed in v3.6.5

**Comment by samayueren** [comment]
Error is still present in 3.6.5
